src/components/NewCommentForm.tsx

```diff
@@ -44,6 +44,14 @@ export const NewCommentForm: React.FC<Props> = ({ onSubmit }) => {
     setErrors(current => ({ ...current, [field]: false }));
   };
 
+  const validateEmail = (value: string) => {
+    if (value.includes('@')) {
+      return true;
+    }
+
+    return false;
+  };
```

Comment:
it would be good to add better validation and simplify the function: 
```suggestion
  const validateEmail = (value: string) => {
    return value.match(/.*\@.*\.\w{2,3}/g);
  };
```